utils/scripts/create_parameter_json_from_nfcore_schema.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/env python3
+
+"""
+Generates a parameter json for AWS HealthOmics workflows
+from NF-CORE nextflow schema JSON.
+"""
+
+from argparse import ArgumentParser
+import json
+from pprint import pprint
+import logging
+
+# Create a logger
+logging.basicConfig(format="%(name)s - %(asctime)s %(levelname)s: %(message)s")
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument('--nfcore_schema_file', type=str, required=True,
+                    help="nextflow schema JSON used in NF-CORE nextflow workflows")
```

Comment:
It would make the app a lot more flexible if the in file and out file args were optional and if not specified then in is STDIN and out is STDOUT. Then you could compose it with pipes.

Reply:
I can add that flexibility but curious in what situation pipes would be recommended